lib/reflection.psi

```diff
@@ -186,6 +240,11 @@ interface PSC::Reflection<> is
         func Sibling_Region(Region) -> optional Region
           is import(#region_sibling_region)
 
+        func Num_Trees(Region) -> Tree_Index
+          is import(#region_num_trees)
+        func Nth_Tree(Region; Tree_Index) -> optional Tree
```

Comment:
It seems like you might want to have a "Tree_Of" operation that takes a Decl and returns its corresponding tree (Module.Tree, Operation.Tree, Type_Decl.Tree, or Obj_Decl.Tree).  Currently the only way you have of reaching a tree is via regions, which seems somewhat limiting, especially if you already have a Decl in your hand.